framework/src/vectorpostprocessors/ElementsAlongLine.C

```diff
@@ -47,9 +47,8 @@ ElementsAlongLine::execute()
   std::vector<Elem *> intersected_elems;
   std::vector<LineSegment> segments;
 
-  MooseSharedPointer<PointLocatorBase> plb = MooseSharedPointer<PointLocatorBase>(_fe_problem.mesh().getMesh().sub_point_locator().release());
-
-  Moose::elementsIntersectedByLine(_start, _end, _fe_problem.mesh(), plb, intersected_elems, segments);
+  UniquePtr<PointLocatorBase> pl = _fe_problem.mesh().getPointLocator();
+  Moose::elementsIntersectedByLine(_start, _end, _fe_problem.mesh(), pl, intersected_elems, segments);
```

Comment:
Which is why I advocated for raw pointer in that case. I know I'm not going to win that argument with you 💩  👖 s though so I'll just complain instead.